### DISCUSSION TITLE
Programmatically open files

### DISCUSSION THREAD

**Discussion by derekzernach** [uptick/react-keyed-file-browser#200] [Q&A]
Is there a way to programmatically open files? My use case is I want to open a number of folders and sub folders on the initial load and am not seeing anything in the docs for this.

**Comment by Heartless49** [comment]
@derekzernach Based on a comment in a previous issue this feature doesn't seem to be currently available unless I'm mistaken.

https://github.com/uptick/react-keyed-file-browser/issues/186#issuecomment-873248869

**Reply by derekzernach** [reply]
Dang, I was able to do it in a previous version by pointing a ref to this component, but am unable to in the latest version.
